config/internal_users.yml

```diff
@@ -10,13 +10,6 @@ _meta:
 
 ## Demo users
 
-admin:
```

Comment:
Does the cluster fail to startup with this change?  Could you include what that looks like?